crates/stackable-operator/src/cli.rs

```diff
@@ -230,17 +232,28 @@ pub enum Command<Run: Args = ProductOperatorRun> {
 #[derive(clap::Parser, Debug, PartialEq, Eq)]
 #[command(long_about = "")]
 pub struct ProductOperatorRun {
+    #[command(flatten)]
+    pub common: CommonStackableCliArgs,
+
+    #[command(flatten)]
+    pub operator_environment: OperatorEnvironmentOptions,
+
     /// Provides the path to a product-config file
     #[arg(long, short = 'p', value_name = "FILE", default_value = "", env)]
     pub product_config: ProductConfigPath,
 
     /// Provides a specific namespace to watch (instead of watching all namespaces)
     #[arg(long, env, default_value = "")]
     pub watch_namespace: WatchNamespace,
+}
 
-    #[command(flatten)]
-    pub operator_environment: OperatorEnvironmentOptions,
-
+/// All the CLI arguments that all (or at least most) Stackable applications use.
+///
+/// [`ProductOperatorRun`] is intended for operators, but it has fields that are not needed for
+/// utilities such as `user-info-fetcher` or `opa-bundle-builder`. So this struct offers a limited
+/// set, that should be shared across all Stackable tools running on Kubernetes.
+#[derive(clap::Parser, Debug, PartialEq, Eq)]
+pub struct CommonStackableCliArgs {
```

Comment:
note: If we decide to go with the term "arguments", we should spell it out. Also get rid of the "Stackable" term.

We also should consider what we name these in general. Currently, there are two terms in use: arguments and options. We should only use one.


```suggestion
pub struct CommonArguments {
```

Reply:
Hmm yeah, we use options in the other places...
Renamed it to `CommonOptions` in 2c57374